app/views/applications/show.html.erb

```diff
@@ -0,0 +1,49 @@
+<h1> Application Details </h1>
+
+
+<% if @application.application_status != "Pending" %>
+  <p><strong>Add a Pet to this Application</strong></p>
+  <%= form_with url: "/applications/#{@application.id}/search_pets", method: :get do |form| %>
+    <%= form.label :name, "Search Pets by Name:" %>
+    <%= form.text_field :name %>
+    <%= form.submit "Search" %>
+  <% end %>
+
+  <% if @search_results %>
+    <% if @search_results.empty? %>
+     <p>No Pet Found. :/ Please try again.</p>
+    <% else %>
+      <% @search_results.each do |pet| %>
+        <p>
+          <%= pet.name %>
+          <%= button_to 'Adopt this Pet', "/applications/#{@application.id}/adopt_pet?pet_id=#{pet.id}", method: :patch %>
+        </p>
+      <% end %>
+    <% end  %>
+  <% end  %>  
+<% end %>
+
+<h3>Applicant Details</h3>
+<p><strong>Applicant Name:</strong> <%= @application.applicant_name %></p>
+<p><strong>Address:</strong> <%= @application.address %></p>
+<p><strong>City:</strong> <%= @application.city %></p>
+<p><strong>State:</strong> <%= @application.state %></p>
+<p><strong>Zip code:</strong> <%= @application.zip_code %></p>
+<p><strong>Why my home is a good fit for a new pet:</strong> <%= @application.description %></p>
+
+
+<p><strong>Applying to adopt:</strong>
+<% @application.pets.each do |pet| %>
+  <%= link_to pet.name, "/pets/#{pet.id}" %>
+<% end %></p>
+
+<% if @application.pets.any? && @application.application_status != "Pending" %>  
```

Comment:
This can definitely be considering display logic so it's okay to include this in the view per MVC convention!

IMO it would also be okay to create a model method to encapsulate it if it were more complex, and name the method something like `submittable?` so that you can call it on the application here to determine if you should allow the submit. This is beneficial because then you would be able to unit test it at the model spec level. But in this case, I don't think this display logic is complex enough to warrant needing to do this